docs/Doxyfile.in

```diff
@@ -61,14 +61,14 @@ PROJECT_BRIEF          = "An easy to use Notecard Library for C/C++"
 # pixels and the maximum width should not exceed 200 pixels. Doxygen will copy
 # the logo to the output directory.
 
-PROJECT_LOGO           = assets/blues-wireless.png
+PROJECT_LOGO           = @NOTE_C_BASE@/assets/blues_logo_no_text.png
 
 # The OUTPUT_DIRECTORY tag is used to specify the (relative or absolute) path
 # into which the generated documentation will be written. If a relative path is
 # entered, it will be relative to the location where doxygen was started. If
 # left blank the current directory will be used.
 
-OUTPUT_DIRECTORY       = .
+OUTPUT_DIRECTORY       = @DOXY_OUT@
```

Comment:
The doxyrunner plugin formats Doxyfile.in to produce Doxyfile, doing variable substitution for variables between `@`. Doxygen then uses Doxyfile to build the docs.